keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -93,40 +93,52 @@ to begin with? AdminNetworkPolicy is a popular alternative.
 
 ## Motivation
 
-Kubernetes provides NetworkPolicy resources to control traffic within a
-cluster. NetworkPolicies focus on expressing a developers intent to secure
-their applications. Thus, in order to satisfy the needs of a security admin,
-we propose to introduce a new API that captures the administrators intent.
+Kubernetes provides the NetworkPolicy resource to control traffic within a
+cluster. NetworkPolicy focuses on expressing a developer's intent to secure
+their applications. However, it was not intended to be used for administrative
+traffic control, which is reflected by its design:
+- It has no concept of explicit "deny" rules, because the application deployer can
```

Comment:
I think it might be more useful to highlight the implicit deny, rather than the lack of explicit deny here.... With the logic thinking being that developers may not always be sure exactly who should access their apps, which makes "playing it safe" with a whitelist model reasonable

However Now with the cluster admin as the persona, It's now safe to assume that they should know the exact security state of their cluster, and therefore should be able to express explicit denies as desired.